framework/src/org/checkerframework/common/value/ValueCheckerUtils.java

```diff
@@ -134,11 +134,10 @@ public static Range getRangeFromValues(List<? extends Number> values) {
     }
 
     /**
-     * Get all possible values from the given type and cast them into Long type, Double type, or
-     * Character type accordingly. Only support casting to integral type and double type.
+     * Get all possible values from the given type and cast them into a boxed primitive type.
      *
      * @param range the given range
-     * @param expectedType the expected type
+     * @param expectedType the expected type (must be a boxed type, not a primitive type)
```

Comment:
Can you make this a separate paragraph and explain, why primitive types don't work?
I guess it's because a `List` can only contain boxed types, but it would be good to make explicit.